src/pke/unittest/UnitTestENCRYPT.cpp

```diff
@@ -109,8 +109,10 @@ static std::vector<TEST_CASE_Encrypt_Decrypt> testCases = {
     { STRING_TEST, "06", {BGVRNS_SCHEME, 256,  2,         59,     DFLT, BATCH,   OPTIMIZED, DFLT,  1,      60,      HEStd_NotSet, BV,     FIXEDAUTO,       DFLT,    256,   DFLT,   DFLT,      0,          DFLT, DFLT,} },
     { STRING_TEST, "07", {BGVRNS_SCHEME, 256,  2,         59,     DFLT, BATCH,   OPTIMIZED, DFLT,  1,      60,      HEStd_NotSet, BV,     FLEXIBLEAUTO,    DFLT,    256,   DFLT,   DFLT,      0,          DFLT, DFLT,} },
     { STRING_TEST, "08", {BGVRNS_SCHEME, 256,  2,         59,     4,    BATCH,   OPTIMIZED, DFLT,  1,      60,      HEStd_NotSet, BV,     FLEXIBLEAUTOEXT, DFLT,    256,   DFLT,   DFLT,      0,          DFLT, DFLT,} },
-    { STRING_TEST, "09", {BFVRNS_SCHEME, DFLT, DFLT,      60,     20,   BATCH,   RLWE,      DFLT,  DFLT,   DFLT,    DFLT,         BV,     FIXEDMANUAL,     DFLT,    256,   DFLT,   DFLT,      2,          DFLT, HPS},  },
-    { STRING_TEST, "10", {BFVRNS_SCHEME, DFLT, DFLT,      60,     20,   BATCH,   OPTIMIZED, DFLT,  DFLT,   DFLT,    DFLT,         BV,     FIXEDMANUAL,     DFLT,    256,   DFLT,   DFLT,      2,          DFLT, BEHZ}, },
+    { STRING_TEST, "09", {BFVRNS_SCHEME, DFLT, DFLT,      60,     20,   BATCH,   RLWE,      DFLT,  DFLT,   DFLT,    DFLT,         BV,     FIXEDMANUAL,     DFLT,    256,   DFLT,   DFLT,      2,          DFLT, HPS},               },
```

Comment:
It feels like unit tests for FIXEDAUTO, FLEXIBLEAUTO and FLEXIBLEAUTOEXT are missing in all files